keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/PersistentEvent.kt

```diff
@@ -15,7 +27,7 @@ abstract class PersistentEvent {
   }
 
   enum class Scope {
-    RESOURCE,
-    APPLICATION
+    @JsonProperty("resource") RESOURCE,
+    @JsonProperty("application") APPLICATION
```

Comment:
What does this do? Those don't look like properties to me, how is that annotation relevant?

Reply:
For enum values, this just uses the value you pass in as the enum value. In other words, just a way to make them lowercase here.

Reply:
From the docs:
> Starting with Jackson 2.6 this annotation may also be used to change serialization of Enum like so:
>
> ```java
> public enum MyEnum {
>     @JsonProperty("theFirstValue") THE_FIRST_VALUE,
>     @JsonProperty("another_value") ANOTHER_VALUE;
> }
> ```
> as an alternative to using `JsonValue` annotation.

Reply:
That's great! We should change all the other places we're doing lowercasing the other way. I'll submit a PR